### DISCUSSION TITLE
When could we have MongoDB in Elsa 3

### DISCUSSION THREAD

**Discussion by michaelmarzouk** [elsa-workflows/elsa-core#4990] [Q&A]
Do you have a scheduling on MongoDB custom persistence on elsa 3?

**Comment by glime-ai** [comment]
Based on the context provided, there is no direct mention of a scheduling feature specifically for MongoDB custom persistence in Elsa 3. However, Elsa Workflows does support persistence, and the persistence mechanism is abstract, allowing for custom persistence providers. This means that you could potentially implement a custom persistence provider for MongoDB that includes scheduling capabilities.

Elsa 3 introduces a simplified persistence abstraction that allows you to choose from a range of persistence technologies, including MongoDB. While the core documentation does not explicitly detail a scheduling feature for MongoDB, you could leverage the flexibility of Elsa's persistence mechanism to implement such a feature if needed.

For more detailed information on how to implement custom persistence providers or to inquire about specific features like scheduling on MongoDB, you may want to explore the official Elsa documentation, check the GitHub repository for issues or discussions related to your query, or reach out to the Elsa community on platforms like Discord or Stack Overflow.

If you are looking for guidance on implementing custom persistence with scheduling for MongoDB in Elsa 3, I would recommend checking the official documentation for any updates on this feature, exploring the GitHub issues for related discussions, or asking for help in the community forums.
